design/FY2019/NFP-AFN-Contaminant-Transport.md

```diff
@@ -0,0 +1,262 @@
+
+```
+AirflowNetwork:MultiZone:Zone,
+      \min-fields 8
+      \extensible:1
+      \memo This object is used to simultaneously control a thermal zone's window and door openings,
+      \memo both exterior and interior.
+  A1, \field Zone Name
+      \required-field
+      \reference AirFlowNetworkMultizoneZones
+      \type object-list
+      \object-list ZoneNames
+      \note Enter the zone name where ventilation control is required.
+
+  ...
+
+  A6, \field Occupant Ventilation Control Name
+      \type object-list
+      \object-list AirflowNetworkOccupantVentilationControlNames
+      \note Enter the name where Occupancy Ventilation Control is required.
+  N7, \field Source Sink Muliplier
+      \type real
+      \default 1.0
+      \note Multiplier to be applied to sources and sinks
+  A7, \field Contaminant Source Sink 1
+      \begin-extensible
+      \type object-list
+      \object-list AirflowNetworkContaminantSourceSinks
+  A8, \field Contaminant Source Sink 2
+      \type object-list
+      \object-list AirflowNetworkMaterialSourceSinks
+  ...
+```
+
+The distribution linkage object will be modified to include a filter component.
+
+```
+AirflowNetwork:Distribution:Linkage,
+      \min-fields 4
+      \extensible:1
+      \memo This object defines the connection between two nodes and a component.
+ A1 , \field Name
+      \required-field
+      \type alpha
+      \note Enter a unique name for this object.
+
+ ...
+
+ A5 , \field Thermal Zone Name
+      \type object-list
+      \object-list ZoneNames
+      \note Only used if component = AirflowNetwork:Distribution:Component:Duct
+      \note The zone name is where AirflowNetwork:Distribution:Component:Duct is exposed. Leave this field blank if the duct
+      \note conduction loss is ignored.
+ A6 , \field Filter 1
+      \begin-extensible
+      \type object-list
+      \object-list AirflowNetworkFilters
+ A7 , \field Filter 2
+      \type object-list
+      \object-list AirflowNetworkFilters
+ ...
+```
+
+The simulation control object will be modified to include a simulation option:
+
+```
+AirflowNetwork:SimulationControl,
+      \min-fields 12
+      \unique-object
+      \memo This object defines the global parameters used in an Airflow Network simulation.
+ A1 , \field Name
+      \required-field
+      \note Enter a unique name for this object.
+
+ ...
+
+ A8 , \field Solver
+      \note Select the solver to use for the pressure network solution
+      \type choice
+      \key SkylineLU
+      \key ConjugateGradient
+      \default SkylineLU
+ A9 ; \field Transport Simulation Type
+      \note The type of transport simulation desired. 
+      \note Selecting None will disable simulation entirely.
+      \choice
+      \key None
+      \key ImplicitEuler
+      \key ExplicitEuler
+      \key CrankNicolson
+      \default None
+```
+
+The new IDD objects are:
+
+* A material object that describes a transported material
+
+```
+AirflowNetwork:Material,
```

Comment:
@mjwitte suggests that this name is too generic. Suggested options:

 * AirborneMaterial - I'm going to veto this one, the materials aren't always airborne
* Constituent - Vetoing this one too, to me this one implies that it is part of something
* Species - This is what CONTAM uses
* Substance - This is the one I like the best, if nobody chimes in and stops me I'm going to go with this one